### PR TITLE
Version bump to coredns 1.8.1

### PR DESCRIPTION
**Pull request by mrueg**
@haad version bump to coredns-1.8.1 https://github.com/coredns/coredns/releases/tag/v1.8.1 https://coredns.io/2021/01/20/coredns-1.8.1-release/

### CODE REVIEW

**Comment by chrisohaver** [comment]
I think the 1.8.1 release image is not multi-arch, which, IIRC, was a requirement from Kubernetes when making CoreDNS the default cluster DNS.

File: stable/coredns/templates/configmap.yaml [hunk 1]
```diff
@@ -12,6 +12,9 @@ metadata:
     kubernetes.io/name: "CoreDNS"
     {{- end }}
     app.kubernetes.io/name: {{ template "coredns.name" . }}
```
Comment (haad) [inline stable/coredns/templates/configmap.yaml:14]: Can we do this change in separate PR ? Mixing them will make it harder to revert if we for some reason need to go back.

**Review by haad — CHANGES_REQUESTED** [review]
I'm ok with version bump, but let's do non related changes in a separate PR.

File: stable/coredns/templates/configmap.yaml (hunk 1 above)
Comment (mrueg) [reply stable/coredns/templates/configmap.yaml:14]: sure, will do.

**Comment by mrueg** [comment]
The RBAC needs also to be updated to include EndpointSlices for it to work.

**Comment by haad** [comment]
Can you update this PR so we can merge it ?
